templates/pulpcore-apache.conf.erb

```diff
@@ -0,0 +1,30 @@
+<VirtualHost *:80>
```

Comment:
Note that we can't do a full vhost here, Foreman on a server defines the vhost and we are dropping a 'fragment' into a directory that gets included into the vhost definition for us. We only really want the proxy pass definitions further down in this template.